src/Microsoft.ML.Data/Data/Conversion.cs

```diff
@@ -17,16 +17,12 @@ namespace Microsoft.ML.Runtime.Data.Conversion
     using BL = DvBool;
     using DT = DvDateTime;
     using DZ = DvDateTimeZone;
-    using I1 = DvInt1;
-    using I2 = DvInt2;
-    using I4 = DvInt4;
-    using I8 = DvInt8;
     using R4 = Single;
     using R8 = Double;
-    using RawI1 = SByte;
-    using RawI2 = Int16;
-    using RawI4 = Int32;
-    using RawI8 = Int64;
+    using I1 = SByte;
```

Comment:
Would it make sense to just start using the C# keywords inline in the code?  That way readers didn't have to map `I2` => `short` in their heads?

Reply:
I think this is generally the case except in this one file, if we consider its nature I feel like this would actually impede readability, at least for me.

---
In reply to: [210964331](https://github.com/dotnet/machinelearning/pull/683#discussion_r210964331) [](ancestors = 210964331)